### PR TITLE
[SG-1469] Handle redirects and banner subtitles for topics

### PR DESCRIPTION
**Pull request by bookworm2000**
For all news view pages, i.e. `sf.gov/news/topics/4`

### CODE REVIEW

**Review by arlina-espinoza — CHANGES_REQUESTED** [review]
Hi @bookworm2000 , this looks good :) We should just also make it so that node create/edit/deletes cascades to this logic, like for news.

File: web/modules/custom/sfgov_utilities/sfgov_utilities.install [hunk 1]
```diff
@@ -56,3 +56,36 @@ function sfgov_utilities_update_8703() {
       }
   }
 }
+
+/**
+ * Add path aliases for topic news pages.
+ */
+function sfgov_utilities_update_8705() {
```
[the remark this answers is not in the document]
Comment (bookworm2000) [reply web/modules/custom/sfgov_utilities/sfgov_utilities.install:63]: Yes, if we merge PR 957 in first which has update_8704

File: web/themes/custom/sfgovpl/includes/views.inc [hunk 2]
```diff
@@ -14,6 +15,23 @@ function sfgovpl_preprocess_views_view__meetings__page(&$variables) {
...
 }
 
+/**
+ * Implements hook_preprocess_views_view()
+ *
+ * Adds a twig template variable for topic news.
+ */
+function sfgovpl_preprocess_views_view__news__page_2(&$variables) {
+  // Retrieve topic id.
+  $nid = !empty($variables['view']->args[0]) ? $variables['view']->args[0]: NULL;
```
[the remark this answers is not in the document]
Comment (bookworm2000) [reply web/themes/custom/sfgovpl/includes/views.inc]: Made this change. Now I need to figure out how to resolve the create/edit/delete logic like in the other ticket (PR 957)

[the remark this answers is not in the document]
Comment (bookworm2000) [reply web/themes/custom/sfgovpl/includes/views.inc]: Integrated the changes with the other PR so now this is working.

**Review by arlina-espinoza — DISMISSED** [review]
LGTM

**Review by aekong — APPROVED** [review]

**Pull request merged**